setup.cfg

```diff
@@ -179,7 +179,7 @@ mssql = mssql+pyodbc://scott:tiger^5HHH@mssql2017:1433/test?driver=ODBC+Driver+1
 mssql_async = mssql+aioodbc://scott:tiger^5HHH@mssql2017:1433/test?driver=ODBC+Driver+18+for+SQL+Server&TrustServerCertificate=yes
 pymssql = mssql+pymssql://scott:tiger^5HHH@mssql2017:1433/test
 docker_mssql = mssql+pyodbc://scott:tiger^5HHH@127.0.0.1:1433/test?driver=ODBC+Driver+18+for+SQL+Server
-oracle = oracle+cx_oracle://scott:tiger@oracle18c/xe
-cxoracle = oracle+cx_oracle://scott:tiger@oracle18c/xe
-oracledb = oracle+oracledb://scott:tiger@oracle18c/xe
-docker_oracle = oracle+cx_oracle://scott:tiger@127.0.0.1:1521/?service_name=XEPDB1
+oracle = oracle+cx_oracle://scott:tiger@oracle23c/free
+cxoracle = oracle+cx_oracle://scott:tiger@oracle23c/free
+oracledb = oracle+oracledb://scott:tiger@oracle23c/free
+docker_oracle = oracle+cx_oracle://scott:tiger@127.0.0.1:1521/?service_name=FREEPDB1
```

Comment:
other than the docker one, the other are used by the ci, so let's revert that bit

Reply:
(at least I think, cc @zzzeek )

Reply:
the setup.cfg entries are biased towards....me personally!   oracle18c is the hostname I have here in our CI environment.   so we can update the README but until I update CI can we please revert the setup.cfg part, thanks!

Reply:
IIRC docker_oracle is fine to update, since it's just something I added when adding the docker example that's been updated here

Reply:
Reverted the CI server entries back to `oracle18c/xe`